blocks/header/header.js

```diff
@@ -117,6 +117,7 @@ export default async function decorate(block) {
   block.textContent = '';
   const nav = document.createElement('nav');
```

Comment:
you can use `createElement` function from common and you can send class names and other properties ;)